docs[translate] translate zh docs from en version in turorial

SisyphusZheng opened the pull request:
## What does this PR do?
This PR adds translated Chinese versions of several tutorial files in the Preact documentation. By providing these translated materials, it makes it easier for Chinese - speaking users to learn Preact through step - by step

## Problem Being Solved
Previously, the Chinese documentation might have lacked detailed and up - to - date tutorial content, which could be a barrier for Chinese users to quickly understand and master Preact. This PR addresses this issue by adding Chinese translations of key tutorial sections, helping Chinese users to learn Preact more effectively without having to rely solely on the English documentation.

## Changes Made
File Addition: Added seven new Markdown files in the directory. These files are translations of the corresponding English tutorial files, including step tutorials 03-09.

Translation Work: Translated all the content in these files from English to Chinese, ensuring that the translated text accurately conveys the original meaning and is easy for Chinese users to understand.
Verification: Followed the contributor guidelines and conducted local builds to verify that the new files are integrated correctly into the documentation system and that there are no formatting or link - related issues.

rschristian commented on content/zh/tutorial/04-state.md:
```diff
@@ -0,0 +1,186 @@
+---
+title: State
```
Nit: super, super minor, but usually the markdown titles should match the first header on the page as it goes to form `document.title`.

If you have a reason for using English titles, no problem (and certainly let me know if this is some issue on our site), else, they probably should be switched to also be translated.

SisyphusZheng commented:
Thank you for your suggestion. I encountered a YAML issue while editing the file and forgot to change the title back during the fix. I will resolve this issue tomorrow. It's already very late now.

发送自 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: Ryan Christian ***@***.***>
发送时间: Sunday, May 18, 2025 3:39:28 AM
收件人: preactjs/preact-www ***@***.***>
抄送: zhizheng ***@***.***>; Author ***@***.***>
主题: Re: [preactjs/preact-www] docs[translate] translate zh docs from en version in turorial (PR #1276)


@rschristian commented on this pull request.

________________________________

In content/zh/tutorial/04-state.md<https://github.com/preactjs/preact-www/pull/1276#discussion_r2094195469>:

> @@ -0,0 +1,186 @@
+---
+title: State


Nit: super, super minor, but usually the markdown titles should match the first header on the page as it goes to form document.title.

If you have a reason for using English titles, no problem (and certainly let me know if this is some issue on our site), else, they probably should be switched to also be translated.

―
Reply to this email directly, view it on GitHub<https://github.com/preactjs/preact-www/pull/1276#pullrequestreview-2848382399>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BC2V34WESNBZZBZWV5IMWBT266F7BAVCNFSM6AAAAAB5K4ZXQOVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDQNBYGM4DEMZZHE>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

SisyphusZheng commented:
--patch-1
## What have I revised?

-Fixed the feedback of 01-vdom.md in the solved state. Now the feedback formats of Chinese tutorial documents and English documents are consistent.
-Modified partial localized titles.

## Why haven’t some titles or English terms been revised?
-Because considering the specificity of terminology, direct translation into Chinese may instead cause confusion for readers.

rschristian reviewed (approved):
Thank you so much! We really appreciate this.

> Because considering the specificity of terminology, direct translation into Chinese may instead cause confusion for readers.

Makes sense, thanks! I just ask as there's _a lot_ of content and it's easy to miss something here or there by accident (I've done it myself quite often). But whatever makes most sense to native speakers is absolutely what we'd like to ship.

SisyphusZheng commented:
> Thank you so much! We really appreciate this.
> 
> 
> 
> > Because considering the specificity of terminology, direct translation into Chinese may instead cause confusion for readers.
> 
> 
> 
> Makes sense, thanks! I just ask as there's _a lot_ of content and it's easy to miss something here or there by accident (I've done it myself quite often). But whatever makes most sense to native speakers is absolutely what we'd like to ship. 

I came here through the Fresh project. Recently, while reviewing the Preactjs documentation, I was pleasantly surprised to find a Chinese version available. However, it's a pity that there are many gaps in the content. I have checked the contributor guidelines and will try to handle and contribute to the Chinese part of the documentation. Of course, if anything I do isn't up to par, please feel free to point it out. Thank you sincerely for your reminders and positive responses.

rschristian commented:
> However, it's a pity that there are many gaps in the content. I have checked the contributor guidelines and will try to handle and contribute to the Chinese part of the documentation.

Unfortunately none of the core team speaks Chinese, nor most of the other languages we support, so we have to rely on community contributions quite heavily for all translations.

The core team & the community certainly appreciates any and all translations offered, every little bit helps a lot. Thanks!
